data/json/mutations/mutations.json

```diff
@@ -427,7 +427,7 @@
     "enchantments": [
       {
         "condition": { "not": "is_day" },
-        "values": [ { "value": "FOOTSTEP_NOISE", "multiply": 0.5 }, { "value": "ATTACK_NOISE", "multiply": 0.8 } ]
+        "values": [ { "value": "FOOTSTEP_NOISE", "multiply": -0.5 }, { "value": "ATTACK_NOISE", "multiply": -0.8 } ]
```

Comment:
I don't know the original intention of the mutation, but I assume it was meant as 50% and 80% of the base value respectively, so ATTACK_NOISE should probably be -0.2 instead.

Reply:
Yeah you are right, 80% seems very excessive.